week-5/A-dom-manipulation/exercise.js

```diff
@@ -4,30 +4,64 @@ Task 1
 
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
+var btn_changeColor = document.querySelector("#bgrChangeBtn");
+btn_changeColor.addEventListener("click", changeBgrColor);
 
-
+function changeBgrColor() {
+  document.querySelector("body").style.backgroundColor = "red";
+}
 
 /*
 Task 2
 ======
 
 When a user clicks the 'ALERT' button, an alert box should pop up with the text "Thanks for visiting Bikes for Refugees!"
 */
+var btn_alert = document.querySelector("#alertbtn");
+btn_alert.addEventListener("click", alertTxt);
 
-
-
+function alertTxt() {
+  alert("Thanks for visiting Bikes for Refugees!");
+}
 /*
 Task 3
 ======
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
+var btn_addText = document.querySelector("#addTextbtn");
+btn_addText.addEventListener("click", addParagraph);
 
+function addParagraph() {
+  var newParagraph = document.createElement("p");
+  var div_withId_jumbotron = document.querySelector(".jumbotron");
 
+  newParagraph.innerText = "Read more below.";
+  newParagraph.style.margin = 10;
 
+  div_withId_jumbotron.appendChild(newParagraph);
+}
 /*
 Task 4
 ======
 
 When the 'Larger links!' button is clicked, the text of all links on the page should increase.
-*/
\ No newline at end of file
+*/
+console.log("Hello");
+
+var btn_largerLink = document.querySelector("#largerLinksBtn");
+btn_largerLink.addEventListener("click", largeLinks);
+
+function largeLinks() {
+  var allLinks = document.querySelectorAll(
```

Comment:
There were probably many answers to this problem. This is the one I used:
```
    document.querySelectorAll("a").forEach((link) => link.style.fontSize = "2em");
```

Reply:
Exactly. I tried to change only the three links that exist on the page and anytime you click on the button their size will be increase.